src/component/Header.js

```diff
@@ -1,17 +1,21 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
+import { FaUserAlt } from 'react-icons/fa';
 
 const Header = () => (
   <div className="header inline-display">
-    <Link to="/"><h2>Bookstore CMS</h2></Link>
-    {' '}
-    <nav>
-      <Link to="/">BOOKS</Link>
+    <div className="home-menu">
+      <Link to="/"><h2>Bookstore CMS</h2></Link>
       {' '}
-      |
-      {' '}
-      <Link to="categories">CATEGORIES</Link>
-    </nav>
+      <nav>
+        <Link to="/">BOOKS</Link>
+        {' '}
+        |
+        {' '}
+        <Link to="categories">CATEGORIES</Link>
+      </nav>
+    </div>
+    <FaUserAlt className="styleIcon" />
```

Comment:
- [ ] please try to make the account icon a little bit small in order to match the design 
- Yours 
![image](https://user-images.githubusercontent.com/61790587/180277413-70505654-08da-48c9-bd2b-fe7fe7bea0d3.png)
- Design 
![image](https://user-images.githubusercontent.com/61790587/180277435-aa582a68-829e-437c-9786-a893348dcd81.png)